eyeshade/workers/referrals.js

```diff
@@ -117,7 +117,7 @@ module.exports.consumer = (runtime) => {
 
     // drop documents that do not start with 'publishers#uuid:' or have a 'removed' id
     const filteredDocs = docs.filter(({ referral: { _id: { owner } } }) => {
-      return !owner || (owner.slice(0, 16) === 'publishers#uuid:' && owner.slice(16) !== 'removed')
+      return !owner || owner.slice(0, 16) === 'publishers#uuid:' || owner.slice(16) !== 'removed'
```

Comment:
Do you think maybe that we should log a message to sentry? It seems like we should know if promo is sending badly formatted values to eyeshade